scripts/release/test-verify-artifacts.py

```
@@ -0,0 +1,632 @@
+        ):
+            with self.subTest(path=path):
+                files = self.evidence_files()
+                files[path] = b""
+                paired_path = path.replace("one", "two")
+                files[paired_path] = b""
+                self.refresh_evidence_manifest(files)
+                self.archive(f"jazz-{VERSION}-benchmark-evidence.tar.gz", files)
+                self.write_checksums()
+                self.assert_rejected("determinism evidence must be nonempty")
+
+    def test_build_alpha_assembles_and_verifies_a_fixture_candidate(self) -> None:
```

Ruff (0.16.1)

[warning] Too many statements (51 > 50)

(PLR0915)